pydatalab/pydatalab/routes/v0_1/items.py

```diff
@@ -74,6 +74,56 @@ def dereference_files(file_ids: List[Union[str, ObjectId]]) -> Dict[str, Dict]:
     return results
 
 
+def get_equipment_summary():
+    if not current_user.is_authenticated and not CONFIG.TESTING:
+        return (
+            jsonify(
+                status="error",
+                message="Authorization required to access chemical inventory.",
+            ),
+            401,
+        )
+
+    _project = {
+        "_id": 0,
+        "creators": {
+            "display_name": 1,
+            "contact_email": 1,
+        },
+        # "collections": {
+        #     "collection_id": 1,
+        #     "title": 1,
+        # },
+        "item_id": 1,
+        "name": 1,
+        "nblocks": {"$size": "$display_order"},
+        "characteristic_chemical_formula": 1,
+        "type": 1,
+        "date": 1,
+        "refcode": 1,
+        "location": 1,
+    }
+
+    items = [
+        doc
+        for doc in flask_mongo.db.items.aggregate(
+            [
+                {
+                    "$match": {
+                        "type": "equipment",
+                        **get_default_permissions(user_only=False),
```

Comment:
Does equipment need to be private?

Reply:
Good point, we can make equipment open to all users (at least for the time being)